docs/modules/reference/pages/provisioning/handlers/vmware.adoc

```diff
@@ -3,6 +3,8 @@
 
 [[vmware-handler]]
 = VMWare Handler
+:description: Learn how to use and configure the VMware adapter to pull hosts and/or virtual machines from a vCenter server into OpenNMS Horizon/Meridian.
+//should this be in the Adapters section? Was it perhaps not moved because of the link concern?
```

Comment:
Adapters change the data of a req.  Handlers build node lists from some other source.
```suggestion
```

Reply:
It is referred to as an adapter throughout this section, though, instead of a handler.